spec/system/assessor_interface/filtering_application_forms_spec.rb

```diff
@@ -37,11 +40,21 @@ def then_i_see_a_list_of_applications_filtered_by_name
     expect(page).to have_content("Cher Bert")
   end
 
+  def and_i_apply_the_country_filter
+    fill_in "Country", with: "France"
+    click_button "Apply filters"
+  end
+
+  def then_i_see_a_list_of_applications_filtered_by_country
+    expect(page).to have_content("Cher Bert")
```

Comment:
Should we create an application specifically for each filter? Just thinking this expectation would pass if we accidentally submitted the previous one's filter inputs.

Reply:
Yep, that's a good point, I've amended the commit with that change.